src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
 [assembly: InternalsVisibleTo("OrleansAWSUtils")]
+[assembly: InternalsVisibleTo("OrleansGoogleUtils")]
```

Comment:
Ok, will check on that. @jason-bragg I'm now on Gitter, when you can, plz ping me there.